### PR TITLE
[1LP][RFR] C&U skip for GCE

### PR DESCRIPTION
**Pull request by digitronik**
Purpose or Intent
=================

- Skipping test for GCE : BZ-1511099

{{pytest: cfme/tests/test_utilization_metrics.py -v --use-provider complete --long-running }}

### CODE REVIEW

**Comment by digitronik** [comment]
Failures are not due to this code. for GCE skipping properly.

File: cfme/tests/test_utilization_metrics.py [hunk 1]
```diff
@@ -170,10 +168,9 @@ def test_raw_metric_vm_memory(metrics_collection, appliance, provider):
 @pytest.mark.rhv2
 @pytest.mark.meta(
     blockers=[BZ(1408963, forced_streams=["5.7", "5.8", "upstream"],
```
[the remark this answers is not in the document]
Comment (rlbabyuk) [reply cfme/tests/test_utilization_metrics.py]: Required: Please update regarding comment above

File: cfme/tests/test_utilization_metrics.py [hunk 2]
```diff
@@ -192,8 +189,8 @@ def test_raw_metric_vm_network(metrics_collection, appliance, provider):
     blockers=[BZ(1322094, forced_streams=["5.6", "5.7"],
```
[the remark this answers is not in the document]
Comment (rlbabyuk) [reply cfme/tests/test_utilization_metrics.py]: Required: Please update regarding comment above

**Comment by digitronik** [comment]
@rlbabyuk added changes. Increase timeout and fix cu-24x7 script.

File: cfme/tests/test_utilization_metrics.py [hunk 3]
```diff
@@ -125,8 +125,8 @@ def query_metric_db(appliance, provider, metric, vm_name=None, host_name=None):
 
 @pytest.mark.rhv2
 # Tests to check that specific metrics are being collected
-@pytest.mark.meta(blockers=[BZ(1525296,
-                               unblock=lambda provider: not provider.one_of(CloudProvider))])
+@pytest.mark.meta(blockers=[BZ(1511099, forced_streams=["5.8", "5.9", "upstream"],
```
Comment (psav) [inline cfme/tests/test_utilization_metrics.py]:
```python
@pytest.mark.meta(
    blockers=[BZ(1511099, forced_streams=["5.8", "5.9", "upstream"],
    unblock=lambda provider: not provider.one_of(GCEProvider))]
)
```

File: cfme/tests/test_utilization_metrics.py [hunk 4]
```diff
@@ -251,8 +251,8 @@ def test_raw_metric_host_network(metrics_collection, appliance, provider):
 @pytest.mark.uncollectif(
     lambda provider: provider.one_of(CloudProvider))
 @pytest.mark.meta(
-    blockers=[BZ(1424589, forced_streams=["5.7", "5.8", "upstream"],
-        unblock=lambda provider: not provider.one_of(RHEVMProvider))]
+    blockers=[BZ(1424589, forced_streams=["5.8", "upstream"],
+    unblock=lambda provider: not provider.one_of(RHEVMProvider))]
 )
```
Comment (nachandr) [inline cfme/tests/test_utilization_metrics.py]: @digitronik , test_raw_metric_host_disk needs to be skipped for RHV in 59 as well.

File: cfme/tests/test_utilization_metrics.py [hunk 5]
```diff
@@ -92,7 +92,7 @@ def metrics_collection(appliance, clean_setup_provider, provider, enable_candu):
         elif provider.category == "infra" and vm_count and host_count:
             return
         else:
-            time.sleep(15)
+            time.sleep(20)
 
     if time.time() > start_time + timeout:
         raise Exception("Timed out waiting for metrics to be collected")
```
Comment (nachandr) [inline cfme/tests/test_utilization_metrics.py]: I know we've had this code in metrics_collection() for sometime now and I think it's time to switch to wait_for :).

Comment (digitronik) [reply cfme/tests/test_utilization_metrics.py]: As per @nachandr and my discussion; we going to refactor C&U codes; At that time we will add those changes.

**Comment by digitronik** [comment]
For 5.8 one test fails due to `vm disk activity` on 5.8. its `cu-24x7 VM` disk script problem not due to code.

File: cfme/tests/test_utilization_metrics.py (hunk 5 above)
Comment (psav) [reply cfme/tests/test_utilization_metrics.py]: **REQUIRED:** Please add a TODO in there so we don't forget

Comment (digitronik) [reply cfme/tests/test_utilization_metrics.py]: @psav added `ToDo: refactor code with wait_for`

**Pull request merged**